kartei.cls

```diff
@@ -327,21 +321,24 @@
   \fi%
   % Build Cover page before starting environment
   %
-\cohead{\color{CardHead}\theCardID}
-\lohead{\color{CardHead}\dasfach{#1}}
-\rohead{\color{CardHead}\derkommentar{#3}}
+\lohead{\theCardID} % VERTAUSCHT Bernd
+\cohead{\dasfach{#1}} % VERTAUSCHT Bernd
+%\cohead{\theCardID} % ORIGINAL Bernd
+%\lohead{\dasfach{#1}} % ORIGINAL Bernd
+\rohead{\derkommentar{#3}}
+\rofoot{Bernd S.E. (berndschelf@gmail.com)} %ERGÄNZT
```

Comment:
Das bitte auch nicht hardcoded in der Klasse sondern wenn mit nem Hook.

Reply:
Jetzt bin ich überfordert. Ich beherrsche die Sprache leider nicht. Ich habe alles mehr durch try&error zurechtgeschustert. Hätte jetzt auch spontan kein guten Vorschlag für ein Vorgehen. Dieser Pull Request hat extrem viel manuelles rumgefrickel.